docs/pipeline.md

```diff
@@ -1,20 +1,21 @@
-# Pipeline
+# Dataset
 
-A Fondant pipeline is a tool for building complex workflows by creating a Directed Acyclic Graph 
-(DAG) of different components that need to be executed. With Fondant, you can use both reusable
-components and custom components, and chain them into a pipeline.
+A Fondant Dataset is a checkpoint in a Directed Acyclic Graph 
+(DAG) of one or more different components that need to be executed. With Fondant, you can use both reusable
+components and custom components, and chain them together.
 
-## Composing a pipeline
 
-Start by creating a `pipeline.py` file and adding the following code.
+[//]: # (TODO update this section once we have the workspace)
```

Comment:
What is the idea of the workspace now?

Reply:
There will be no workspace. We will use a working_directory to store all the execution artifacts. We will update the documentation in a separate PR.